book/06-github/sections/2-contributing.asc

```diff
@@ -110,126 +110,126 @@ To https://github.com/tonychacon/blink
+Si vous fusionnez cette branche dans `master` et poussez le tout sur GitHub, la requête de tirage sera fermée automatiquement.
 
-C'est le flux de travail de base que la plupart des projets GitHub utilisent.
-Les branches thématiques sont créées, les requêtes de tirages sont ouvertes dessus, une discussion s'engage, du travail additionnel peut être ajouté sur la branche et à la fin, la requête est soit fermée, soit fusionnée.
+C'est le processus de travail de base que la plupart des projets GitHub utilisent.
+Des branches thématiques sont créées, des requêtes de tirage sont ouvertes dessus, une discussion s'engage, du travail additionnel peut être ajouté sur la branche et à la fin, la requête est soit fermée, soit fusionnée.
 
 [NOTE]
-.Pas seulement avec des dépôts répliqués
+.Pas seulement avec des dépôts clonés
 ====
 Il est important de noter que vous pouvez aussi ouvrir une requête de tirage entre deux branches du même dépôt.
-Si vous travaillez sur une fonctionnalité avec quelqu'un et que vous avez tous deux accès en écriture au projet, vous pouvez pousser une branche thématique sur le dépôt et ouvrir un requête de tirage dessus vers la branche `master` de ce même projet pour démarrer une revue de code et une discussion.
-Aucune duplication n'est nécessaire.
+Si vous travaillez sur une fonctionnalité avec quelqu'un et que vous avez tous deux accès en écriture au projet, vous pouvez pousser une branche thématique sur le dépôt et ouvrir une requête de tirage dessus vers la branche `master` de ce même projet pour démarrer une revue de code et une discussion.
+Aucun clonage n'est nécessaire.
 ====
 
-==== _Pull requests_ avancées
+==== Requêtes de tirage avancées
 
-Après avoir traité les bases de la contribution à un projet sur GitHub, voyons quelques trucs et astuces d'efficacité concernant les requêtes de tirage.
+Après avoir présenté les bases de la contribution à un projet sur GitHub, voyons quelques trucs et astuces concernant les requêtes de tirage afin d'améliorer votre efficacité .
 
-===== _Pull requests_ comme patches
+===== Requêtes de tirage comme patchs
 
-Il est important de comprendre que pour de nombreux projets, les requêtes de tirage sont vues comme des piles de patchs parfaits qui doivent s'appliquer dans l'ordre, ce que la plupart de projets dont les interactions reposent sur une liste de diffusion voient comme une série de patchs envoyés par mail.
-La plupart des projets GitHub pensent les branches de requête de tirage comme des conversations itératives autour d'une modification proposée, aboutissant à un diff unifié qui est appliqué par fusion.
+Il est important de comprendre que pour de nombreux projets, les requêtes de tirage ne sont pas vues comme des files d'attente de patchs parfaits qui doivent s'appliquer correctement dans l'ordre, alors que la plupart des projets basés sur des listes de diffusion fonctionnent par série de patchs envoyés par courriel.
```

Comment:
"It’s important to understand that many projects don’t really think of Pull Requests as queues of perfect patches that should apply cleanly in order, as most mailing list-based projects think of patch series contributions. Most GitHub projects think about Pull Request branches as iterative conversations around a proposed change, culminating in a unified diff that is applied by merging."

Quand je relis l'anglais, je comprends que dans les projets "classiques" (via email), on fonctionne par patch alors qu'avec github les PR ne sont pas équivalent à la procédure par patchs, ils servent à travailler ensemble et quand c'est fini, on merge le point final et pas la suite de modif.